src/QueryParameters/GetTasksParameters.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Spatie\FloatSdk\QueryParameters;
+
+use Spatie\FloatSdk\Concerns\HasPaginationAndSort;
+
+class GetTasksParameters
+{
+    use HasPaginationAndSort;
+
+    /**
+     * @param  array<string>|null  $fields
+     */
+    public function __construct(
+        public ?int $projectId = null,
```

Comment:
I think we'll also need the task_id

Reply:
We can add a separate request for this: https://developer.float.com/api_reference.html#!/Project_Tasks/getProjectTask